changelog/5233.trivial.rst

```diff
@@ -0,0 +1 @@
+Added tests to report back status of remote urls pinged during test run.
```

Comment:
```suggestion
Online tests can now report back status of remote urls and will XFAIL if the remote server is done.
```
Would this be correct?

Reply:
How about this?

Online tests can now report back status of remote urls and will XFAIL if any remote server is pinged.

Reply:
If they are pinged unsuccessfully?

Reply:
I think there is some misunderstanding here,
- the `remote_status` test function will ping each url and currently will FAIL if the server is unreachable.
- during the intercept session, if any test function tries to connect to remote server then it would XFAIL

Reply:
Ah I see. When I opened the original issue and now, I guess my thinking has shifted.

I would like an online test to XFAIL if a remote server is down, avoiding it taking down our remote test suite check. 
 Tho I feel like this might be a better change to make in the pytest-remotedata package instead.